hedera-node/src/test/java/com/hedera/services/store/tokens/HederaTokenStoreTest.java

```diff
@@ -262,7 +263,33 @@
 					.setFixedFee(CustomFeesOuterClass.FixedFee.newBuilder()
 							.setAmount(0L)
 					)).build();
-
+	private CustomFeesOuterClass.CustomFees grpcMaxLessThanMinCustomFees = CustomFeesOuterClass.CustomFees.newBuilder()
+			.addCustomFees(CustomFeesOuterClass.CustomFee.newBuilder()
+					.setFeeCollectorAccountId(feeCollector)
+					.setFractionalFee(CustomFeesOuterClass.FractionalFee.newBuilder()
+							.setFractionalAmount(Fraction.newBuilder()
+									.setNumerator(123L).setDenominator(1_000L))
+							.setMaximumAmount(2L)
+							.setMinimumAmount(10L)
+					)).build();
+	private CustomFeesOuterClass.CustomFees grpcMaxEqualToMinCustomFees = CustomFeesOuterClass.CustomFees.newBuilder()
+			.addCustomFees(CustomFeesOuterClass.CustomFee.newBuilder()
+					.setFeeCollectorAccountId(feeCollector)
+					.setFractionalFee(CustomFeesOuterClass.FractionalFee.newBuilder()
+							.setFractionalAmount(Fraction.newBuilder()
+									.setNumerator(123L).setDenominator(1_000L))
+							.setMaximumAmount(2L)
+							.setMinimumAmount(2L)
+							.setMinimumAmount(2L)
```

Comment:
`setMinimumAmount(2L)` is repeated twice here

Reply:
Though it won't hurt, but may cause some confusion.

Reply:
@abhishek-hedera  @ljianghedera  Thank you ! It is removed